src/analysisd/labels.c

```diff
@@ -82,16 +82,21 @@ wlabel_t* labels_find(const Eventinfo *lf) {
             // data structure for the first time in the labels cache. We need to release
             // the memory and request the labels from cache again.
             mdebug2("Labels already in cache for agent %s. Updating.", lf->agent_id);
-            free(data);
+            os_free(data);
```

Comment:
`os_free` frees the pointer and sets it to `NULL`. We do not need that because `OSHash_Get` writes that variable just below.

Reply:
Fixed